particles/PipeApps2/PipeApps.recipes

```diff
@@ -1,6 +1,6 @@
 import 'MapsAutofill.recipes'
 import 'PeopleAutofill.recipes'
-import 'ArtistAutofill.recipes'
-import 'Caption.recipes'
-import 'UpdateMe.recipes'
-import 'ArtistPipe.recipes'
+// import 'ArtistAutofill.recipes'
```

Comment:
Is this intentional?

Reply:
thanks, uncommented (and got rid of canonical.recipes - Scott says it is redundant)